examples/ipython/colored_christoffel_symbols.ipynb

```diff
@@ -81,7 +81,7 @@
     {
      "data": {
       "text/latex": [
-       "$\\displaystyle g =\\left[\\begin{matrix}1 & 0 & 0\\\\0 & 1 & 0\\\\0 & 0 & 1\\end{matrix}\\right]$"
+       "$\\displaystyle g =\\left[\\begin{array}{ccc}1 & 0 & 0\\\\0 & 1 & 0\\\\0 & 0 & 1\\end{array}\\right]$"
```

Comment:
These changes are caused by swapping to the default sympy printer to our printer.